Add basic docs for py-terminal (#933)
pyscript/pyscript: docs/reference/plugins/py-terminal.md

# &lt;py-terminal&gt;

> This is one of the core plugins in PyScript, which is active by default. With it, you can receive `stdout` and `stderr` from your python code, which will be displayed on the page in `<py-terminal>`.

Reviewer: ```suggestion
This is one of the core plugins in PyScript, which is active by default. With it, you can print to `stdout` and `stderr` from your python code, and the output will be displayed on the page in `<py-terminal>`.
```

## Configuration

You can control how `<py-terminal>` behaves by setting the value of the  `terminal` configuration in your `<py-config>`.

| value | description |
|-------|-------------|

> | `false` | Deactivate the plugin, no `<py-terminal>` will be added to the page |

Reviewer: ```suggestion
| `false` | Don't add `<py-terminal>` to the page |
```

the plugin is still "active", in the sense that you can still place a `<py-terminal>` manually and it works.

> | `true` | Adds a `<py-terminal>` to the page even if nothing is shown in the terminal yet. |

Reviewer: ```suggestion
| `true` | Automatically add a `<py-terminal>` to the page |
```

> | `"auto"` | This is the default value for the `<py-terminal>`, the element will be shown on the page once there is something to show |

Reviewer: ```suggestion
| `"auto"` | This is the default. Automatically add a `<py-terminal auto>`, to the page. The terminal is initially hidden and automatically shown as soon as something writes to `stdout` and/or `stderr |
```

### Examples

```html
<py-config>
    terminal = true
</py-config>

<py-script>
    print("Hello, world!")
</py-script>
```

> This example will create a new `<py-terminal>`, the value "Hello, world!" that was printed will show in it.

Reviewer: I would also add a note which explains that you can place a `<py-terminal>` manually (and you can place even multiple ones)

# &lt;py-terminal&gt;

This is one of the core plugins in PyScript, which is active by default. With it, you can print to `stdout` and `stderr` from your python code, and the output will be displayed on the page in `<py-terminal>`.

## Configuration

You can control how `<py-terminal>` behaves by setting the value of the  `terminal` configuration in your `<py-config>`.

| value | description |
|-------|-------------|
| `false` | Don't add `<py-terminal>` to the page |
| `true` | Automatically add a `<py-terminal>` to the page |

> | `"auto"` | This is the default. Automatically add a `<py-terminal auto>`, to the page. The terminal is initially hidden and automacially shown as soon as something writes to `stdout` and/or `stderr` |

Reviewer: typo :)

```suggestion
| `"auto"` | This is the default. Automatically add a `<py-terminal auto>`, to the page. The terminal is initially hidden and automatically shown as soon as something writes to `stdout` and/or `stderr` |
```

Author: Great catch thank you!

```

This example will create a new `<py-terminal>`, the value "Hello, world!" that was printed will show in it.

You can also add one (or more) `<py-terminal>` to the page manually.

```html
<py-config>
    terminal = true

> </py-config>

Author: Oh gotcha!
